test/JDBC/expected/Test_parallel_query.out

```diff
@@ -0,0 +1,367 @@
+-- tsql
+CREATE LOGIN l1 WITH PASSWORD = '12345678'
+GO
+
+CREATE USER l1
+GO
+
+CREATE TABLE pq_t (id INT)
```

Comment:
It is not clear if all test cases are covered, can you put one line comment for each test to describe what use case it covers like
-- Skip permission check for temp table in BBF on parallel worker
-- Skip permission check for PG temp table in BBF on parallel worker
-- Permission check error for PG temp table without parallel worker and so on